testnets/xiontestnet/chain.json

```diff
@@ -267,21 +265,21 @@
         "ibc_go_version": "v7.4.0",
         "consensus": {
           "type": "cometbft",
-          "version": "0.37.4"
+          "version": "v0.37.4"
         },
         "binaries": {
           "linux/amd64": "https://github.com/burnt-labs/xion/releases/download/v7.0.0/xiond-v7.0.0-linux-amd64:checksum=sha256:e7431453c3e7adadb7d75516d6cdc27e0ab4e9bdbdb620f7af4697a72c0c71bb"
         },
         "sdk": {
           "type": "cosmos",
-          "version": "0.47.10"
+          "version": "v0.47.10"
```

Comment:
corresponding with this value update, the `"cosmos_sdk_version"` property value must also be updated to the same value (within version 7).